src/utils/colonyMotions.ts

```diff
@@ -1,5 +1,10 @@
 import { defineMessage } from 'react-intl';
 
+export enum MotionVote {
+  NAY = 0,
+  YAY = 1,
+}
```

Comment:
I think this will cause a merge conflict since it was added on some other PR as well.

But leave it in, as we can deal with the merge conflict later.